packages/vuetify/src/services/theme/index.ts

```diff
@@ -179,6 +179,22 @@ export class Theme extends Service {
     this.vueMeta = true
 
     const meta = root.$meta()
+
+    const metaAdd = typeof meta.addApp === 'function'
+    if (metaAdd) {
+      const { set } = root.$meta().addApp('vuetify')
+
+      set({
+        style: [{
+          cssText: this.generatedStyles,
```

Comment:
You could probably also set [`once: true`](https://vue-meta.nuxtjs.org/api/#once), but you probably shouldnt call `.set` multiple times with the same stylsheet then because it will probably be added each time then

If thats an issue you could set a prop on `$root` to check if its already added or not

Reply:
The old code would check if the stylesheet is already there and if so only re-apply the `cssText`:
https://github.com/vuetifyjs/vuetify/blob/06419697cd3ba33ff116397d272ebda902706beb/packages/vuetify/src/services/theme/index.ts#L204
I assume this only happens during SSR, but you are saying we should just skip it instead?

Reply:
> If thats an issue you could set a prop on `$root` to check if its already added or not

is there no `getApp()` available? 😅

Reply:
If that check is done only for SSR it should be possible to add a [`[tagidkeyname]`](https://vue-meta.nuxtjs.org/api/#tagidkeyname) prop so its only added once. Ofc you have to omit `once` in that case.

Reply:
How does it work ? I'm not sure to understand how it fixes thing, it makes double places where we set styles.
Also with the `addApp` I also don't understand how `this.generatedStyles` gonna be reactive if not in a function. Please give me your light 😁

Reply:
My remarks are generic of nature. Havent look at the vuetify specifics